src/data/items/foodBartending.js

```diff
@@ -0,0 +1,355 @@
+    },
+    drinkAle: {
+        name: "Mag-Ale",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/ale.png")        
+    }, */
+    drinkWhiskey: {
+        name: "Whiskey",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/whiskey.png")
+    },
+    drinkVodka: {
+        name: "Vodka",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/vodka.png")
+    },
+    /* drinkAbsinthe: {
+        name: "Absinthe",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/absinthe.png")
+    },
+    drinkCognac: {
+        name: "Cognac",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/cognac.png")
+    }, */
+    drinkGin: {
+        name: "Gin",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/gin.png")
+    },
+    /* drinkKahlua: {
+        name: "Kahlua",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/kahlua.png")
+    }, */
+    drinkRum: {
+        name: "Rum",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/rum.png")
+    },
+    /* drinkSake: {
+        name: "Sake",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/sake.png")
+    }, */
+    drinkTequila: {
+        name: "Tequila",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/tequila.png")
+    },
+    drinkLemonJuice: {
+        name: "Lemon Juice",
+        sellPrice: -1,
+        icon: require("@/assets/art/bartending/lemonjuice.png")
+    },
+    drinkOrangeJuice: {
+        name: "Orange Juice",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/orangejuice.png"),
+        healAmount: 35
+    },
+    drinkTomatoJuice: {
+        name: "Tomato Juice",
+        sellPrice: -2,
+        icon: require("@/assets/art/bartending/tomatojuice.png"),
+        healAmount: 15
+    },
+    drinkCream: {
+        name: "Cream",
+        sellPrice: -1,
+        icon: require("@/assets/art/bartending/cream.png")
+    },
+    drinkSpacecola: {
+        name: "Space Cola",
+        sellPrice: 1,
+        icon: require("@/assets/art/bartending/spacecola.png")
+    },
+    drinkBlackRussian: {
+        name: "Black Russian",
+        sellPrice: 50,
+        icon: require("@/assets/art/bartending/blackrussian.png"),
+        healAmount: 15
+    },
+    drinkWhiteRussian: {
+        name: "White Russian",
+        sellPrice: 95,
+        icon: require("@/assets/art/bartending/whiterussian.png"),
+        healAmount: 25
+    },
+    drinkBilk: {
+        name: "Bilk",
+        sellPrice: 5,
+        icon: require("@/assets/art/bartending/bilk.png"),
+        healAmount: 6
+    },
+    drinkTripleSec: {
+        name: "Triple Sec",
+        sellPrice: 12,
+        icon: require("@/assets/art/bartending/triplesec.png"),
+        healAmount: 10,
+        stats: {
+            precision: -1,
+            evasion: -1,
+            power: 1
+        }
+    },
+    drinkAtomicBomb: {
```

Comment:
I have a concern that the price to make this is too cheap (```-50xp```), and will need to be increased.